### PR TITLE
get chat and contact colors

### PR DESCRIPTION
**Pull request by r10s**
this pr adds two functions that return "fine" colors for chats and contacts.
the colors can be used for drawing avatars or the names above messages in group chats.

### CODE REVIEW

File: src/dc_chat.c [hunk 1]
```diff
@@ -259,6 +259,46 @@ char* dc_chat_get_profile_image(const dc_chat_t* chat)
...
+ * For 1:1 chats, the color is calculated from the contact's email address.
+ * Otherwise, the chat name is used.
+ * The color can be used for an fallback avatar with white initials
+ * as well as for headlines in bubbles of group chats.
+ *
+ * @memberof dc_chat_t
+ * @param chat The chat object.
+ * @return Color as 0x00rrggbb with rr=red, gg=green, bb=blue
+ *     each in the range 0-255.
+ */
```
Comment (ralphtheninja) [inline src/dc_chat.c]: You mean from `0` to `ff`?

**Comment by violoncelloch** [comment]
And as background color for the message bubbles?

File: src/dc_chat.c (hunk 1 above)
Comment (r10s) [reply src/dc_chat.c]: jupp :)
do you think we should better write it in hex?

**Comment by r10s** [comment]
> And as background color for the message bubbles?

might be possible, however, the plan is to use white/light backgrounds in the bubbles.

File: src/dc_chat.c (hunk 1 above)
Comment (ralphtheninja) [reply src/dc_chat.c]: Nah, it's probably fine :)

**Pull request merged**